Sources/UI/ActionButton/ActionButton.swift

```diff
@@ -33,25 +39,56 @@ public struct ActionButton: View {
   }
   
   public var body: some View {
-    GeometryReader { geo in
-      ZStack(alignment: getAlignment(for: properties.extraImage)) {
-        ButtonView(size: geo.size, action: buttonAction, properties: properties)
-        ExtraImageView(imageAlignment: properties.extraImage, tintColor: properties.textColor, size: geo.size)
+    Button(action: {
+      properties.actionHandler?()
+    }) {
+      HStack {
+        properties.leftImage?.image
+          .resizable()
+          .aspectRatio(contentMode: .fit)
+          .frame(width: properties.leftImage?.size.width,
+                 height: properties.leftImage?.size.height)
+        Spacer()
+        HStack {
+          properties.titleLeftImage?.image
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            .frame(width: properties.titleLeftImage?.size.width,
+                   height: properties.titleLeftImage?.size.height)
+          Text(properties.title)
+            .font(properties.font)
+          properties.titleRightImage?.image
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            .frame(width: properties.titleRightImage?.size.width,
+                   height: properties.titleRightImage?.size.height)
+        }
+        Spacer()
+        properties.rightImage?.image
+          .resizable()
+          .aspectRatio(contentMode: .fit)
+          .frame(width: properties.rightImage?.size.width,
+                 height: properties.rightImage?.size.height)
       }
+      .padding()
```

Comment:
Should we add a property in the ViewModel to control padding also?

Reply:
The reason why I kept it this way is because default padding works really good for the customization types that we have.
According to documentation:   _If you set the value to nil, SwiftUI uses a platform-specific default amount_.
Either way I don't have any strong feelings about this.